tests/manager/models/test_etcd.py

```diff
@@ -22,38 +21,16 @@
 """
 
 
-@pytest.fixture(scope="module")
-def client() -> Client:
-    return Client(Schema(query=Queries, mutation=Mutations, auto_camelcase=False))
-
-
-@pytest.fixture(scope="module")
```

Comment:
Since the context function in this file is used as read-only, shouldn't the scope be set to module?